bicytok/distanceMetricFuncs.py

```diff
@@ -20,13 +20,10 @@ def KL_EMD_1D(
         KL_div_vals: vector of KL Divergences per receptor
         EMD_vals: a vector of EMDs per receptor
     """
-
     assert all(
-        isinstance(i, bool) for i in np.append(targ, offTarg)
+        isinstance(i, np.bool) for i in np.append(targ, offTarg)
```

Comment:
I think after v2 NumPy just wants you to use builtin bool?

Reply:
I thought so too but I have been having issues with matching types with bool vs np.bool. For some reason the entry types are np.bool and will not match with builtin bool.